python/graphstorm/model/utils.py

```diff
@@ -125,8 +125,67 @@ def _get_sparse_emb_range(num_embs, local_rank, world_size):
         end = num_embs if local_rank + 1 == world_size else end
     return start, end
 
+def save_sparse_emb(model_path, sparse_emb, ntype,
+                    local_rank, world_size):
+    """ Save sparse emb `sparse_emb` into `model_path`
+
+        Sparse embeddings are stored as:
+            $model_path/$ntype/sparse_emb_0.pt
+            $model_path/$ntype/sparse_emb_1.pt
+            ...
+
+        Example:
+        --------
+        Save sparse embeddings of an input embed_layer by calling ``save_sparse_emb``
+        iterating all the sparse embeddings of the embed_layer
+
+        .. code::
+            # embed_layer is the input embed_layer
+            embed_layer = embed_layer.module \
+                if isinstance(embed_layer, DistributedDataParallel) \
+                else embed_layer
+
+            for ntype, sparse_emb in embed_layer.sparse_embeds.items():
+                save_sparse_emb(model_path, sparse_emb,
+                                ntype, local_rank, world_size)
+
+        Parameters
+        ----------
+        model_path: str
+            The path of the model is saved.
+        sparse_emb: dgl.distributed.DistEmbedding
+            A Distributed node embedding.
+        ntype: str
+            The node type the embedding belongs to.
+        local_rank : int
```

Comment:
Can we copy and paste the explanation of the local_rank and world_size to here too?